internal/printer/conditions_test.go

```diff
@@ -0,0 +1,75 @@
+package printer
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/vmware-tanzu/octant/internal/testutil"
+	"github.com/vmware-tanzu/octant/pkg/view/component"
+)
+
+func Test_createConditionsTableErrs(t *testing.T) {
+	// No status found
```

Comment:
Any strong opinion on shifting to something like?
```
	cases := []struct {
		name   string
		status unstructured.Unstructured
		err    error
		isOk   bool
		table  *component.Table
	}{
		{
			name:   "no status",
			status: unstructured.Unstructured{Object: map[string]interface{}{"noStatus": nil}},
		},
		{
			name:   "bad status",
			status: unstructured.Unstructured{Object: map[string]interface{}{"status": 1}},
			err:    fmt.Errorf(".status accessor error: 1 is of the type int, expected map[string]interface{}"),
		},
		{
			name:   "no conditions",
			status: unstructured.Unstructured{Object: map[string]interface{}{"status": map[string]interface{}{"noConditions": nil}}},
			table:  component.NewTable("Conditions", "There are no conditions!", nil),
		},
	}

	for _, tc := range cases {
		t.Run(tc.name, func(t *testing.T) {
			table, ok, err := createConditionsTable(tc.status, "", nil)
			if tc.err == nil {
				require.NoError(t, err)
			} else {
				require.Errorf(t, tc.err, ".status accessor error: 1 is of the type int, expected map[string]interface{}", err)
			}
			require.Equal(t, tc.isOk, ok)
			component.AssertEqual(t, tc.table, table)
		})
	}
```

Reply:
Since you already coded it up, no strong opinion at all. :)